lib/simplifyExpression/fractionsSearch/addConstantFractions.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
-const math = require('mathjs');
 const clone = require('../../util/clone');
 const divideByGCD = require('./divideByGCD');
+const math = require('mathjs');
```

Comment:
I think this require should go separate at the top, with a new line after it

because it's a require from outside of `mathsteps`